blocks/hero-banner/hero-banner.css

```diff
@@ -3,6 +3,10 @@
     margin-bottom: 2rem;
 }
 
+.hero-banner.careers {
+    background-image: url("https://main--sunstar--hlxsites.hlx.live/assets/careers-mobile-top.jpg");
```

Comment:
Can relative link work here? If yes, probably we should add it.

Reply:
Thanks good point! I made this change now.